types/pi-pir-sensor/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for pi-pir-sensor 0.0
+// Project: https://github.com/opedromiranda/pi-pir-sensor
+// Definitions by: Mario Murrent <https://github.com/meecodebymariomurrent>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare function Sensor(configuration: Sensor.SensorConfiguration): void;
+
+declare namespace Sensor {
+    class Sensor {
+        lastMovement: Date;
+
+        constructor(sensorConfiguration: SensorConfiguration);
+
+        start(): void;
+
+        stop(): void;
```

Comment:
```suggestion
        stop(): boolean;
```
https://github.com/opedromiranda/pi-pir-sensor/blob/0b8d7d2bd2d153883c6c4e88fa55cacd4f396db3/lib/sensor.js#L49-L58